### PR TITLE
SDK changes for new PlaceOrderEvent

### PR DESCRIPTION
**Pull request by filipzeta**

### CODE REVIEW

File: src/idl/zeta.json [hunk 1]
```diff
@@ -4619,5 +4639,8 @@
       "name": "InvalidTagLength",
       "msg": "Invalid order tag length, too long"
     }
-  ]
+  ],
```
Comment (filipzeta) [inline src/idl/zeta.json:4642]: Is this OK? :/ not sure what it is

Comment (xjcaa) [reply src/idl/zeta.json:4642]: You can re run `anchor build` and `npm run build-sdk` to remove this metadata.

This is attached after you run `anchor test` successfully.

We don't want this 😛

**Review by xjcaa — APPROVED** [review]
Looks beautiful to me.

**Comment by filipzeta** [comment]
> Looks beautiful to me.

look away from the mirror king

**Pull request merged**